app/models.py

```diff
@@ -12,110 +28,45 @@ def strip_services_wrapper(content):
-                            }
-                        }
-                        }
-                    )
-    return res
-
-
-def search_with_lot_filter(query, lot, start_from=0, size=10):
-    res = es.search(index="services",
-                    body={
-                        "from": start_from, "size": size,
-                        "fields": ["id", "serviceName", "lot",
-                                   "serviceSummary"],
-                        "query": {
-                            "filtered": {
-                                "query": {
-                                    "multi_match": {
-                                        "query": query,
-                                        "fields": ["serviceName",
-                                                   "serviceSummary",
-                                                   "serviceFeatures",
-                                                   "serviceBenefits"],
-                                        "operator": "or"
-                                    }
-                                },
-                                "filter": {
-                                    "term": {"lot": lot.lower()}
-                                }
-                            }
-                        }
-                    }
-                    )
-    return res
+def search_without_filters(query):
+    payload = {'q': query}
+    response = requests.get(
+        search_url,
+        params=payload,
+        headers={
+            "authorization": "Bearer {}".format(search_access_token)
+        }
+    )
+    return response.content
 
 
-def search_with_filters(query, args, start_from=0, size=10):
-    filters = []
-    for k, v in args.iteritems():
-        filters.append({"term": {k: v.lower()}})
-    res = es.search(index="services",
-                    body={
-                        "from": start_from, "size": size,
-                        "fields": ["id", "serviceName", "lot",
-                                   "serviceSummary"],
-                        "query": {
-                            "filtered": {
-                                "query": {
-                                    "multi_match": {
-                                        "query": query,
-                                        "fields": ["serviceName",
-                                                   "serviceSummary",
-                                                   "serviceFeatures",
-                                                   "serviceBenefits"],
-                                        "operator": "or"
-                                    }
-                                },
-                                "filter": {
-                                    "bool": {"must": filters}
-                                }
-                            }
-                        }
-                    }
-                    )
-    return res
+def search_with_filters(query, filters):
```

Comment:
You could probably get rid of the `search_without_query` method if you set a default value for the `filter` argument on this method (eg `filters=[]`)